Header header

uint32 image_seq_
time image_stamp_

uint32 descriptor_count_
uint32 descriptor_length_
float32[] data_
